### ISSUE TITLE
JavaScript rather than page refresh

### ISSUE THREAD

**Issue by DeniseColbert** [FreeUKGen/FreeBMD2#717]
From Alison re. accessibility of sorting results:

However, a keyboard-only user would expect the sort to be a button, even though it looks more like a link – because it will do something – rather than the link that it is. A link should go somewhere. It is important because it affects selection: the Space key selects a button; Enter selects a link. See below for my suggested edit to the text in the grey box.

Using VO (VoiceOver) in Safari, the recommended browser, selecting a sort link in a header cell reloads the whole page and the user is returned to the top of the page; one has to use the rotor (the group of listings), for instance, to return to the beginning of the table headers. Very awkward; could we make this better?

**Comment by nurifreembd** [comment]
test with keyboard and screen reader

**Comment by nurifreembd** [comment]
@Vino-S @richardofsussex hello, is there a reason when sorting why the labels are linked and not just the up/down arrow button on the results page please? The users should click the up/down image (looks like a button too) to sort, is that possible? I think only the button should be focused and highlighted and not the labels. @AlOneill do we need a new button if you think the up/down image is not sufficient?


<img width="124" height="131" alt="Image" src="https://github.com/user-attachments/assets/b3fcbd6b-0858-4df4-9132-89466962a032" />

**Comment by AlOneill** [comment]
@nurifreembd 
Thanks for asking. The proposed button will be fine as long as it: 
- is identified programmatically for a screen reader
- has clear visibility when focused, including when tabbed into
- can be activated by keyboard alone, using both Enter and the Spacebar
- can be moved away from, i.e. there is no keyboard trap introduced by the JavaScript

This is all required for WCAG level A, so we must comply if we are aiming for level AA.
